### DISCUSSION TITLE
Postgre - External IDE

### DISCUSSION THREAD

**Discussion by rcsilva81** [supabase/supabase#919] [Questions] [answered]
Hi Guys, is it possible to access my supabase DB using an external IDE, i.e: DBeaver, VS Code?
Tks,
Robert

**Comment by steve-chavez** [comment, marked as answer]
Hey @rcsilva81,

Yes, you can copy paste the postgres connection string located on `/settings/database`(click cog then database on the UI) into any IDE.

**Reply by rcsilva81** [reply]
Thanks.. reason I asked is because I couldn't connect earlier today with my conn details and pass, so I thought I might be blocked. 
But thanks Steve, I'll try again.
